### PR TITLE
Community tab

### PR DESCRIPTION
**Pull request by lacker**
This changes the "support" tab into the "community" tab and moves the current "community resources" section there, along with keeping the original "support" page. We can change the content of the "community" stuff later, this PR is just handling the layout.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ $ cd react/docs
 $ bundle exec rake
 $ bundle exec rake fetch_remotes
 $ bundle exec jekyll serve -w
-$ open http://localhost:4000/react/
+$ open http://localhost:4000/react/index.html
```
Comment (spicyj) [inline docs/README.md]: Why the change?

Comment (lacker) [inline docs/README.md]: You can't actually go to localhost:4000/react, that 404s

File: docs/support.md [hunk 2]
```diff
@@ -1,8 +1,9 @@
 ---
-title: Need help?
-layout: single
 id: support
-permalink: support.html
+title: Where To Get Support
+layout: community
+permalink: community/support.html
```
Comment (zpao) [inline docs/support.md]: Moving this URL but none of the rest that are going into the section?

Should we just go ahead and rename files to fit now as well?

**Comment by lacker** [comment]
screencapture-localhost-4000-react-community-support-html-1475527394177
Here's what the "community" tab looks like now. Basically the same as current "support" page but with the community stuff on the sidebar

**Comment by spicyj** [comment]
Can you get the tab at top to light up properly like the others do?

File: docs/support.md (hunk 2 above)
Comment (lacker) [inline docs/support.md]: Yeah good point I'll rename

**Comment by lacker** [comment]
OK - @spicyj the tab at the top now lights up appropriately, @zpao the files are named according to their URLs, any other feedback from folks?

File: docs/README.md (hunk 1 above)
Comment (spicyj) [inline docs/README.md]: Works for me…

image

**Comment by zpao** [comment]
A couple of these pages are translated (even if not entirely up to date). Should we move those files as well since we moved the other files without change? It's not clear what the final state will be here - it might make sense to delete translations now but if we're going to keep what we have, even if just for now, we should move those files to stay alongside the English content.

**Comment by lacker** [comment]
@zpao OK I moved the translations too.

@spicyj Ah I was actually doing `/react` which 404s, and `/react/` works. If you don't mind let's keep it this way?

**Comment by zpao** [comment]
👍 squash & merge to your heart's content
